foundation/meeting-2023-11-22.md

```diff
@@ -0,0 +1,54 @@
+- Initial discussion on which issues the MonoGame Foundation would like to fund first.
+- Technology highlights:
+    - Microsoft GDKX / GDK Support.
+    - Native AOT.
+    - Vulkan Support. (iOS/Android/MacOS?)
+    - DX12.
+    - Move dependencies within MonoGame for dedicated support, e.g.FAudio and FreeImage.
+- Restoration of Unit Tests for Windows and Mac, including Graphical tests.
+- Review of current Documentation / Tutorials estate and how to do better.
+- New Content design and build Support.
+- Upgrading to DotNet 8 and the move awway from DotNet 6.
+
+## Outcomes
+
+This has been a heavy week for discussions, debates and some real outcomes that will directly benefit all MonoGame Developers.  The foundation has really listened to the community and driven the key priorities needed to move MonoGame forward in the immediate future, with definitive roadmap for ongoing development.
+
+Here is a summary of the outcomes from our week long debate:
+
+- Funding will be international and the MonoGame Foundation has setup an acccount with Wise for international payments.  PayPal will be available as a last resort should developers have issues receiving funds.
+- A schedule for contracts (or Issue Bounties) will be drawn up over the next few weeks/months to support necessary developments (although still the Foundation is offering their services freely to get criticval tasks moving), more on this space later.
+- An overall proposal format will be drafted for any bounties to be offered to paid work for the Foundation.  Principle rule is that no one "profits" from work, the funds are there to help support the work to be completed.
+- The RoadMap, work has begun in earnest and a formal roadmap will be published soon, but here are the highlights for the 3.X cycle:
+    - Native AOT support for cross platform builds.
+    - The addition of Vulkan support (which will also provide Metal support) for iOS/Android. (see [This issue](https://github.com/MonoGame/MonoGame/issues/4593) for details.)
+    - Update Xbox platform for DX 12 and GDKX support (with plans to bring DX12 to Windows once complete).
+    - Unite the Native API for MonoGame for all platforms, to make maintenance and new platforms easier.
+- The content pipeline overhaul - Following increasing feedback regarding the Content Pipeline, the intention is to migrate the existing GUI tool to simply use a DotNet library instead, removing the need to maintain an external tool.  There will also be some internal updates to improve efficiency.  **Note* this will not change how the COntent Pipeline works in your projects, so there are no braking changes (although a Content Pipeline configuration will need to be migrated to the new pattern).
+- Dependencies - The intention is to bring ALL dependencies under MonoGame control (either as direct forks or taking ownership), this will improve the overall reliability of components that MonoGame is heavily dependent on.

```

Comment:
We are just trying to bring the building of them under our control, not the actual dev on them, so we can more easily update them, ensure they are statically linked, etc.